plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -325,6 +385,43 @@ def upload_topoconfig_file(ufm_port, payload):
     else:
         return response.json, SUCCESS_CODE
 
+def get_local_cable_validation_report(ufm_port):
+    '''
+    Send request for cable validation report locally
+    '''
+    ufm_host = "127.0.0.1:{}".format(ufm_port)
+    ufm_protocol = "http"
+    resource = "/plugin/cablevalidation/cablevalidation/report/validation"
+    headers = {"X-Remote-User": "ufmsystem"}
+    response = get_request(ufm_host, ufm_protocol, resource, headers)
+    if not response:
+        return {}, ERROR_CODE
+    else:
+        return json.loads(response.text), SUCCESS_CODE
+
+
+def get_cable_validation_report(cable_validation_server_address,
+                                cable_validation_request_port,
+                                cable_validation_username,
+                                cable_validation_password):
+    '''
+    Send request for cable validation report
+    :param cable_validation_server_address:
+    :param cable_validation_request_port:
+    :param cable_validation_username:
+    :param cable_validation_password:
+    '''
+    # 1 create cookies file 
+    # 2. using cookies file get cable validation report
+    response = post_request_with_cookies(cable_validation_server_address,
```

Comment:
why do we need to create a session on each get request? the session should be created once and use it on each time

Reply:
this is how NDT plugin implemented